cmd/agent/agent.go

```diff
@@ -245,3 +245,21 @@ func loop(c *cli.Context) error {
 	wg.Wait()
 	return nil
 }
+
+func stringSliceAddToMap(sl []string, m map[string]string) error {
+	if m == nil {
+		m = make(map[string]string)
+	}
+	for _, v := range sl {
+		parts := strings.SplitN(v, "=", 2)
+		switch len(parts) {
+		case 2:
+			m[parts[0]] = parts[1]
+		case 1:
+			return fmt.Errorf("key '%s' does not have a value assigned", parts[0])
+		default:
+			return fmt.Errorf("empty string in slice")
```

Comment:
What would happen if someone uses `region=desize=large` (forgets the comma)?

Reply:
it should: `region`= `desize=large`

Reply:
Should we throw an error in that case as well as it will probably be an error?